client/src/main/java/com/aws/greengrass/cli/commands/DeploymentCommand.java

```diff
@@ -137,8 +143,31 @@ public DeploymentCommand(NucleusAdapterIpc nucleusAdapterIpc) {
     public int status(@CommandLine.Option(names = {"-i", "--deploymentId"}, paramLabel = "Deployment ID",
             required = true) String deploymentId) {
 
-        LocalDeployment status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
-        System.out.printf("%s: %s%n", status.getDeploymentId(), status.getStatus());
+        LocalDeploymentStatus status = nucleusAdapterIpc.getLocalDeploymentStatus(deploymentId);
```

Comment:
General question: why don't we extract this whole section to a separate method like - deploymentStatusAnalyst ? Seems a pretty static section.

Reply:
Which section do you mean?

Reply:
From line 146 to 169

Reply:
That piece of code is specific to deployment status command, ie, send status request, receive, process and output response. Not really sure if that would be considered static or if we would achieve any benefit from extracting this section into a different method..

Reply:
Synced offline, we decide to keep this